src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowUpdate.cs

```diff
@@ -39,13 +39,79 @@ public RowUpdate(long rowId, ResultSet associatedResultSet, IEditTableMetadata a
         }
 
         /// <summary>
+        /// Sort order property. Sorts to same position as RowCreate
+        /// </summary>
+        protected override int SortId => 1;
```

Comment:
I really only needed deletes to go last. I haven't yet found a reason to do row create or row update first, so I just set them to the same values. We can revisit if we find it to be problematic.